frontends/web/src/routes/account/summary/coinbalance.tsx

```diff
@@ -70,21 +92,18 @@ export function CoinBalance ({
             </tr>
           </thead>
           <tbody>
-            { accounts.length > 0 ? (
-              coins.map(coinCode => {
-                if (accountsPerCoin[coinCode]?.length >= 1) {
-                  const account = accountsPerCoin[coinCode][0];
-                  return (
-                    <SubTotalCoinRow
-                      key={account.coinCode}
-                      coinCode={account.coinCode}
-                      coinName={account.coinName}
-                      balance={coinsBalances && coinsBalances[coinCode]}
-                    />
-                  );
-                }
-                return null;
-              })) : null}
+            { (Object.keys(coinsWithLightning).length > 0) ? (
+              Object.entries(coinsWithLightning).map(([coinCode, coinName]) => {
```

Comment:
When reading this I could understand what the intention was (iterative over coins with at least 1 account or more), but now I find it hard to understand what the intention is.

Reply:
the intention is to iterate over `coinsWithLightning` but only if there are any keys in it so it won't iterate if it is empty